packages/design-system/src/interfaces/banner.d.ts

```diff
@@ -0,0 +1,13 @@
+import { MouseEventHandler } from "react";
+
+export type BannerSize = 'small' | 'medium' |'large';
+export type BannerType = 'info' | 'warning' | 'success' | 'error'
+
+export interface IBannerProps extends React.HTMLAttributes<HTMLDivElement> {
+  onClose?: MouseEventHandler<HTMLButtonElement>;
+  size?: BannerSize;
+  children: React.ReactNode;
+  type?: BannerType;
```

Comment:
According to propTypes the type should be required.

```suggestion
  type: BannerType;
```

Reply:
✅